src/utility_windows.go

```diff
@@ -242,3 +242,19 @@ func getLocalUser(userName string) (shared.LocalUser, error) {
 	}
 	return shared.LocalUser{}, nil
 }
+
+func getLocalServiceStatus(serviceName string) (shared.Service, error) {
+    serviceDataList, err := wapi.GetServices()
+    var serviceStatusData shared.Service
+	if err != nil {
+        failPrint("Couldn't get local service: " + err.Error())
+        return serviceStatusData, err
```

Comment:
return nil, err

Reply:
cant return nil cus it's not shared.service type

Reply:
nil can be returned regardless of the return type

Reply:
![image](https://user-images.githubusercontent.com/52616034/90677038-a93ead80-e211-11ea-9cf3-e828a4e56afd.png)